.coafile

```diff
@@ -35,5 +35,4 @@ max_lines_per_file = 1000
 enabled = False
 bears = KeywordBear
 
-ci_keywords, keywords_case_insensitive = \#TODO, \# TODO, \#FIXME, \# FIXME
-cs_keywords =
+keywords = \#TODO, \# TODO, \#FIXME, \# FIXME
```

Comment:
this change woud mean that we can't use the latest coala release for this coafile anymore. Do we want that?